dashboard/test/ui/runner.rb

```diff
@@ -321,7 +321,9 @@ def output_synopsis(output_text)
 
     message += "<br/><i>rerun: ./runner.rb -c #{browser_name} -f #{feature} --html</i>"
     HipChat.log message, color: 'red'
-    HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
+    if CDO.hip_chat_logging && !(['adhoc', 'production'].include? Rails.env)
```

Comment:
Good catch Josh. In that case we can just do:
```
HipChat.developers short_message, color: 'red' if Rails.env.test?
```
Since we don't need to log to the developer room on UI test failures from staging or developer environments either.